### PR TITLE
data: Sankhadip-Roy

### PR DESCRIPTION
**Pull request by Sankhadip-Roy**
LinkedIn link fixed

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8211"></a>

### CODE REVIEW

**Review by accodes21 — COMMENTED** [review]
Looks good but you need to solve the merging conflict.

**Comment by Sankhadip-Roy** [comment]
"type": "personal",
 is it needed?

**Comment by Sankhadip-Roy** [comment]
> Looks good but you need to solve the merging conflict.



> Looks good but you need to solve the merging conflict.

"type": "personal",
is it needed?

File: data/Sankhadip-Roy.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "Sankhadip Roy",
+  "type": "personal",
```
Comment (kumarsonsoff3) [inline data/Sankhadip-Roy.json:3]: `type` key is no longer required. Please remove this from your `json`.

```suggestion
```

**Review by Sindhuinti — APPROVED** [review]

**Review by SaraJaoude — APPROVED** [review]

**Pull request merged**